### PR TITLE
libwebm 1.0.0.27

### PR DESCRIPTION
**Pull request by fran6co**

### CODE REVIEW

File: Library/Formula/libwebm.rb [hunk 1]
```diff
@@ -11,13 +11,17 @@ class Libwebm < Formula
...
   end
 
-  def install
-    system "make"
+  depends_on "cmake" => :build
 
-    lib.install "libwebm.a"
+  def install
+    mkdir "macbuild" do
+      system "cmake", *std_cmake_args, ".."
```
Comment (bfontaine) [inline Library/Formula/libwebm.rb]: You can’t add arguments after `*something`:

```
Error: problem in /usr/local/Library/Formula/libwebm.rb
compile error
/usr/local/Library/Formula/libwebm.rb:18: syntax error, unexpected tSTRING_BEG, expecting tAMPER
      system "cmake", *std_cmake_args, ".."
                                        ^
```

Comment (DomT4) [inline Library/Formula/libwebm.rb]: `system "cmake", "..", *std_cmake_args` is the standard here and avoids such problems.

Comment (fran6co) [inline Library/Formula/libwebm.rb]: Fixed

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/963c8cee133914962831e47538a94d6dec6c065c. Thank you for another contribution to Homebrew @fran6co! :smiley_cat:

**Pull request closed without merging**